core/src/bmi.jl

```diff
@@ -494,9 +508,26 @@ BMI.get_time_step(model::Model) = get_proposed_dt(model.integrator)
 
 run(config_file::AbstractString)::Model = run(parsefile(config_file))
 
+function is_current_module(log)
+    (log._module == @__MODULE__) || (parentmodule(log._module) == @__MODULE__)
+end
+
 function run(config::Config)::Model
-    model = BMI.initialize(Model, config)
-    solve!(model.integrator)
-    BMI.finalize(model)
-    return model
+    logger = current_logger()
+
+    # Reconfigure the logger if necessary with the correct loglevel
+    # but make sure to only log from Ribasim
```

Comment:
Why did you only want logging from Ribasim? For debugging I guess sometimes you might also want all the info from DifferentialEquations.

Reply:
Will make a new issue for this.